app/forms/EducationQuestionnaireForm.scala

```diff
@@ -93,20 +93,19 @@ object EducationQuestionnaireForm {
 
       def getOptionalUniversityList(isCandidateCivilServant: String): List[Question] = {
```

Comment:
It is used below:

(isCandidateCivilServant, haveDegree) match  {
 -          case ("Yes", Some("Yes")) => List(